pkg/services/alerting/extractor.go

```diff
@@ -111,6 +118,10 @@ func (e *DashAlertExtractor) getAlertFromPanels(jsonWithPanels *simplejson.Json,
 		rawFor := jsonAlert.Get("for").MustString()
 		var forValue time.Duration
 		if rawFor != "" {
+			unit := unitFormatRegex.FindAllString(rawFor, 1)[0]
```

Comment:
This field was not part of the referenced issue or was it? Wouldn't time.ParseDuration catch any problem with this?

Reply:
I think it is better to distinguish the errors, meaning that if it has happened because of a wrong unit, then throw `ErrWrongUnitFormat.Error`, if something else happens, then throw `"Could not parse for"`. My pov though, feel free to disagree.

Reply:
Yeah I agree. Maybe some better error messages to be done in another ticket 👍

Reply:
Think it's better to address in this PR

Reply:
Oh, okay I've misunderstood your comment:

> Sounds good. However, you wouldn't know for what field this error happened which the failure of time.ParseDuration at least includes.

Will add a better message there :) Thanks!